### DISCUSSION TITLE
Tarik data dari ODBC

### DISCUSSION THREAD

**Discussion by joejnt** [teamalgoritma/community#124] [Q&A] [answered]
Dear Tim Algoritma,

Mohon bantuannya saya punya data dari server bentuk filenya ODBC, tapi saya punya masalah gimana caranya untuk ambil datanya saat menggunakan Rmarkdown. capture data terlampir.

link: 
https://docs.google.com/document/d/1mNyPW_yXCnrKh5k0VngjSKRdF1d_MIPe/edit?usp=sharing&ouid=113694006631251271686&rtpof=true&sd=true

Best regards,
JT

**Comment by joejnt** [comment, marked as answer]
saya coba jalankan fungsi

dbGetQuery('SELECT * FROM ZC0001')

hasilnya error
Error in (function (classes, fdef, mtable) : 
unable to find an inherited method for function ‘dbGetQuery’ for signature ‘"character", "missing"’

**Reply by inytss** [reply]
Dear @joejnt ,
Untuk melakukan fetching data, Bapak dapat menyertakan object connectionnya, pada kasus ini, object connection Bapak adalah `db`.
Saya juga menemukan artikel yang dapat Bapak eksplor lebih lanjut yaitu pada link berikut https://db.rstudio.com/r-packages/odbc/
Pada link tersebut, tersebutkan untuk melakukan fetching data Bapak dapat menggunakan command `dbSenQuery(conn, statement)` dan meneruskannya dengan `dbFetch()` dimana

- `conn`: object connection dengan database
- `statement`: query nya

Sebagai contoh Bapak dapat menggunakan command berikut:
```
table1 <- dbSendQuery(conn = db, statement = "SELECT * FROM ZC0001")
df <- dbFetch(table1)
```

Demikian jawaban dari saya, apabila Bapak merasa jawaban saya membantu silahkan dapat menekan tombol "Mark as answered" pada jawaban ini. Terima kasih :)

**Reply by joejnt** [reply]
Dear @inytss 

Noted, terima kasih untuk insight nya..hasilnya sukses.

Best regards,
JT
